watertap/costing/watertap_costing_package.py

```diff
@@ -153,6 +358,65 @@ def add_specific_electrical_carbon_intensity(
                 doc=f"Specific electrical carbon intensity based on flow {flow_rate.name}",
             ),
         )
+        self._add_flow_component_breakdowns(
+            "electricity",
+            name,
+            flow_rate,
+            period=pyo.units.hr,
+            utilization_factor=1.0,
+            multiplier=self.electrical_carbon_intensity,
+        )
+
+    def _add_flow_component_breakdowns(
+        self,
+        flow_name,
+        name,
+        flow_rate,
+        period=None,
+        utilization_factor=None,
+        multiplier=1.0,
```

Comment:
I don't see this used anywhere in this method... am I missing it?

Reply:
That was a bug -- thanks. Fixed in 82bd46b.